src/accidental.ts

```diff
@@ -297,7 +318,7 @@ export class Accidental extends Modifier {
   }
 
   // Helper function to determine whether two lines of accidentals collide vertically
-  static checkCollision(line1, line2) {
+  static checkCollision(line1?: ILine, line2?: ILine) {
```

Comment:
this is because of the spread operator on line 188:
```javascript
.every((lines: ILine[]) => !this.checkCollision(...lines));
```

typescript compiler suggests it will desctructure an array to 0...n parameters. In our case it is exactly 2, so I should probably change the 188 as follows:
```javascript
.every(([line1, line2]) => !this.checkCollision(line1, line2));
```